### ISSUE TITLE
TextField and other button gesture conflict

### ISSUE THREAD

**Issue by figbin** [flutter/flutter#37912]
TextField have default text, when I tap TextField and focus on, keyboard appear, now DO NOT input any things, try to press RaisedButton want to execute send action but not affect.
contrary the TextField get the gesture and show text selection "paste / select all"

tested device: iPhone 6 plus / iPhone XS
requirements:
1、TextField must have default value: TextEditingController(text: 'mmm...initialize text');
2、TextField on focus don't input any thing
3、tap any other button, the issue appear

![](https://images2.imgbox.com/58/21/hoGO8sSE_o.gif)

**Comment by figbin** [comment]
flutter doctor 
```
[✓] Flutter (Channel beta, v1.7.8+hotfix.4, on Mac OS X 10.14.6 18G87, locale zh-Hans-CN)
    • Flutter version 1.7.8+hotfix.4 at /Users/development/flutter
    • Framework revision 20e59316b8 (3 weeks ago), 2019-07-18 20:04:33 -0700
    • Engine revision fee001c93f
    • Dart version 2.4.0
```

**Comment by CescCesc** [comment]
The problem is like this, when initializing you assign the value to the controller and trigger the TextField input event that accidentally happens to repeat it somewhere several times. TextField will understand it's a long press event. To resolve and avoid this problem, I recommend that you initialize aCtrl in the beginning and assign it to the initState function like this: aCtrl.text = 'mmm ... initialize text';
Hope it helps you !!!

**Comment by CescCesc** [comment]
![image](https://user-images.githubusercontent.com/28478203/62753836-76f35f80-ba97-11e9-81af-7e228f8d3acb.png)

**Comment by figbin** [comment]
Thanks for the respond. I have used your code but still have the problem.

**Comment by figbin** [comment]
fixed on Channel beta, v1.8.3
